Optimizing the Security of your Skywallet.md

```diff
@@ -0,0 +1,48 @@
+
+However, this is not the case for a Seed, and any hacker/malware which gets access to your Seed of the Skywallet can remotely access or steal your funds.
+
+So when it comes to security optimizations, these are the two vital components to be considered.
+
+## PIN Code
+
+Your PIN code is a four-digit combination, and a brand new Skywallet which you receive via shipment does not have a PIN code.
+
+Hence, it is essential first to set a PIN code right after you receive your Skywallet. If you want to know more about how to set the PIN code and personalize your Skywallet, please visit the user manual [Personalization of your Skywallet](https://github.com/skycoin/hardware-wallet/wiki/Getting-to-know-the-wallet).
+
+Though the PIN code is a four-digit combination, it would not be easy for an attacker/hacker to try the different combinations and work it out. 
+
+Because after each failed attempt, one has to wait for a specific time before attempting the next combination and with each failed attempt the waiting time increases exponentially, making it impossible for an attacker/hacker to obtain the PIN code that way.
+
+However, such a security precaution is ineffective if you create the pin as an easy to guess one. 
+
+Examples include but not limited to, important dates, or sequences (like 1234) or repeating digits(1111).
+
+Hence a secure PIN code is something easy to remember and at the same time not easy to guess for a third party.
+
+## Seed
+
+A Seed of the Skywallet is even more critical than a PIN code, as a hacker/malware without access to your Skywallet physically, can remotely access or steal your fund just using the Seed.
+
+As the Seed is not created by you and is a generated one by the device, you need not worry about the strength of the Seed as such.
+
+**However, it is imperative to keep the backup of the Seed in a safe and accessible place, as it is the only way to access your funds if anything happens to your Skywallet.**
+
+Following are some recommendations to backup and store your Seed safely:
+
+* **Do not Store the Seed in an electronic device. Because storing a Seed in an electronic device effectively nullifies the security it ensures, as the electronic device is hackable by a potential hacker and can get access to your Seed.**
+Hence it is recommended to store the Seed in a paper or a notebook which can then be stored in a safe place that is inaccessible to any unauthorized person at the same time not prone to any accidents.<br />
+**The only instance when the Seed is entered into a computer is when you are restoring the backup of your Wallet**, in that case, the Skywallet and Desktop Wallet process the Seed in a secure way which ensures that a third party will not gain access to your Seed.<br />
+The process is explained in detail in the section (Restore Backup)[https://github.com/skycoin/hardware-wallet/wiki/Restore-configuration].
+
+* Create multiple copies of the Seed and store it in separate locations.
```

Comment:
Are these recommendations made by  Trezor?

Reply:
The first 3 are actually recommended by Trezor, however I took the last point from the existing wiki. I felt that is also a valid option, however, if you feel that would confuse the user, I can scrap that last point.

Reply:
No. Should be fine. Thanks for explaining.